makedebian.pl

```diff
@@ -115,6 +117,7 @@
 Priority: optional
 Architecture: all
 Depends: $deps
+Recommends: $recommends
```

Comment:
Is the `Recommends` tag supported on all modern Debian and Ubuntu versions? I seem to recall shying away from it in the past due to lack of support..

Reply:
Yes, it does. It worked even on Debian 9.

Reply:
Recommends has been supported in deb packages for many years. Like 10+ years, at least, but probably longer. It's fine in debs. It's much more recent in RPM.